api/queries_comments.go

```diff
@@ -99,3 +100,30 @@ func CommentsForPullRequest(client *Client, repo ghrepo.Interface, pr *PullReque
 
 	return &Comments{Nodes: comments, TotalCount: len(comments)}, nil
 }
+
+func CommentCreate(client *Client, repoHost string, params map[string]string) (string, error) {
```

Comment:
If this method is only to be used in `pkg/cmd/issue` and `pkg/cmd/pr` packages, consider placing it in `pkg/cmd/issue/shared/http.go` instead. I feel like we should shrink down the API package to only have generic API functionality, and specific queries are implemented in individual packages that need them.

Reply:
I'd like to think about this more so I am going to leave it in place for this PR and then address it when working on commenting for PRs.